### DISCUSSION TITLE
Default allDay to true if no time is specified in an event

### DISCUSSION THREAD

**Discussion by Inirdin** [davish/obsidian-full-calendar#386] [Feature Requests]
It would be nice if the calendar could pull from entire vault and show notes as events which have "Date:: [[2022-02-22]]"
_Title_ is name of the note, if there is no _endDate_ then the event takes place on that single date and if there isn't specified time then _allDay_ is true. I find it really redundant to have all this specified in the note for all the whole day events.

I know, this is multiple request in one. First is pulling from entire vault, then parsing dataview's syntax with "::" outside of front matter and then defaulting to values which aren't specified. Just throwing it here.

**Comment by davish** [comment]
Thanks for submitting this! I think there's three requests in this issue:
1. Pull events from the whole vault
2. Parse event title for dataview fields
3. Default to allDay=true if no time fields are present.

Request #1 is already possible by setting the vault root as a calendar. Unfortunately, request #2 isn't possible because `:` is not a valid character in a filename, and therefore it's not valid in a note title. See #41 for more context here. Request #3 is a good request, and I'll rename this issue to reflect this.

Going forward, please submit separate issues for each feature request. I'll only be tracking the request that is mentioned in the title.
